### ISSUE TITLE
Supported hardware

### ISSUE THREAD

**Issue by michalelektryk** [dguerizec/grbl-Mega-4axis#7]
Hi, what kind of mega shield I'll have to use to make it run?
Are you using rams 1.4 or some custom pcb?

**Comment by aes92000** [comment]
I made my own pcb based on a grbl shield.
I had a few professionally made.


I supports 6 axis and total of 6 output relay controls
Cool & mist (M7, M8, M9)
R1 to R4 (M85, M86, M87, M88)
All toggle on off and  M89 all R? Off

I will send you a board if give me address to mail to.
I will solder everything on my board but you will need the mega.
I will send the Grbl firmware I use also.

I use bCNC to run my machine

**Comment by michalelektryk** [comment]
Thanks! I'll be very glad to play with your board and hopefully finish my 4 axis cutter.

Bielszowicka 64,
Warszawa 04-738
Poland

bCNC has been very useful but I'll probably switch to full custom on PC side.

**Comment by aes92000** [comment]
Another note...
I use all external motor drivers, not the DRV modules.
Will that be an issue?
If not, give me a few days to get put together and copy files, because of
the holidays.
I'll get mailed out and let you know when I do.

Hope it will be very useful.
I have been running board for 4+ years now

**Comment by michalelektryk** [comment]
Can you elaborate a bit on external motor drivers? Does it mean your board has just step/dir outputs routed to external motor controllers?

**Comment by aes92000** [comment]
Correct.

**Comment by michalelektryk** [comment]
Then I'm afraid it'll be of little use to me.

If I can ask you used it with external powerful drivers for big steppers / BLDCs or you simply had board with stepstics on it?

**Comment by aes92000** [comment]
I did start with the small DRV modules but kept burning up.
I am using NEMA23 motors for all 5 axis

I believe I may have an add-on board for the small drivers.
Basically wire as Step, Dir and power from mega board to add-on DVR drivers.
How many axis are you using?

Let me see what I have and I'll see if you can use it.

I am working now but will be back in a couple hours and I will check if I
have what you need.

**Comment by michalelektryk** [comment]
I'm doing 4 axis on 6 motors (2 axis have 2 motors in parallel and that's stretching what's possible on DVRs so anything than can handle a bit more is welcome change).

is any of the 6 relay outputs capable of PWM like standard grbl?

**Comment by aes92000** [comment]
No, relays are on/off only

The dual motors can be parallel from wiring off mega also.
Each on its own DRV and same signal from mega.

If i have 2 add-on boards with 4 motors per board, that would allow dual
motor wiring

**Comment by michalelektryk** [comment]
That'd be perfect. If not then single add-on board and motor windings in parallel should also be sufficient.

**Comment by Oakmanor1** [comment]
just looking at projects

**Comment by michalelektryk** [comment]
thanks

**Comment by michalelektryk** [comment]
Were you able to find anything?

**Comment by aes92000** [comment]
Yes, I actually made you 2 adaptor boards
I'm am in process of soldering the mega shield.

**Comment by aes92000** [comment]
I will send you papers with wiring diagrams also

**Comment by aes92000** [comment]
I have all the boards ready to ship.
It will take me a few days to get mailed because of my work schedule.
I will let you know when the ship.

I am also sending a thumb drive that contains the diagrams, pin outs,
arduino library and pre-compiled .hex files.

**Comment by michalelektryk** [comment]
Sorry, my github somehow supressed notifications.
thank you very much for all your help! I already started reading all docs of GRBL to hopefully be at least close to getting into your code when everything arrives.

**Comment by michalelektryk** [comment]
Hi, how's shipping going?

**Comment by aes92000** [comment]
Wow, I apologize.
I have been out of town working.
I will be back tomorrow and I will get mailed

**Comment by aes92000** [comment]
Ok, slight setback...
I didn't realize the shipping was so darn expensive.

I need to know how you want to proceed.


USPS has NO TRACKING AT ALL

UPS has TRACKING ALL THE WAY

**Comment by michalelektryk** [comment]
I think no tracking. As far as I remember shipping costs UPS may cost few times more than entire controller

**Comment by aes92000** [comment]
Boards on the way
Estimated 7-21 days

**Comment by michalelektryk** [comment]
Thank you very much. I will keep you updated

**Comment by aes92000** [comment]
Need anything to help understand or setup, let me know.

When parts arrive, look on USB for diagrams and GRBL setup files.

**Comment by aes92000** [comment]
get delivery yet???

**Comment by michalelektryk** [comment]
No, still fighting with customs. They can't believe I got such "industrial equipment" as a git. But should hopefully be solved within days

**Comment by aes92000** [comment]
Wow.
I hope it works out soon.
Good luck.

**Comment by michalelektryk** [comment]
Hi,
boards arrived. I hope I'll find some time next week to play with them

**Comment by aes92000** [comment]
Just let me know if I can help in any way
